Restructuring of expansion content section & Battlefield Combat Rules

DerSchlupf opened the pull request:
New Links and Descriptions added; Unit minituares added; fanmade expansion stuff mentioned. Battlefield Rules added with clarifications below the current layout.

TO-DO:
- [ ] improve all 4 assets in battlefield combat rules
- [ ] complete empowered ability token heading (text, example, assets)
- [ ] proofreading of all that new stuff
- [ ] discuss the idea of linking to fanmade stuff (and if yes, then add QR-Codes)
- [ ] idea: Add a Miniature Art asset near " Unit Miniatures" (instead of earth tome)

CURRENT LAYOUT
<img width="635" height="903" alt="image" src="https://github.com/user-attachments/assets/3ffcfe9a-49bd-45cb-862a-950edde6d8a2" />
<img width="1290" height="886" alt="image" src="https://github.com/user-attachments/assets/e6c4a775-7dfa-4e51-a391-f39d9798a12d" />
<img width="1292" height="908" alt="image" src="https://github.com/user-attachments/assets/63345601-56dd-4228-a2fe-f585ad53e1c8" />
<img width="1294" height="897" alt="image" src="https://github.com/user-attachments/assets/3bb819ab-1915-45f0-826b-14f459929180" />
<img width="638" height="905" alt="image" src="https://github.com/user-attachments/assets/589b49f4-54c6-469b-bf9c-a2f73b03b2ec" />



CLARIFICATIONS ABOUT BATTLEFIELD RULES:

**1. Retreat in neutral combats with battlefield boar**d (Source: https://discord.com/channels/740870068178649108/1211591995328299038/1257247721186918462)
<img width="871" height="499" alt="image" src="https://github.com/user-attachments/assets/58a7789f-a05b-4969-a547-d3ca4df1f4a2" />

**2.  Place all obstacles.** Havent found an official statement, but does this count as clarification :D
<img width="747" height="193" alt="image" src="https://github.com/user-attachments/assets/1e59bba1-2d93-47e4-8ede-71eb6e3474dd" />
(Source: https://boardgamegeek.com/thread/3227959/article/44166113#44166113)

**3. How many units on the battlefield map?**
<img width="994" height="189" alt="image" src="https://github.com/user-attachments/assets/64482e7b-1a1a-44f2-b01a-b0a65ed2d67b" />
(Source: https://boardgamegeek.com/thread/3310428/article/44360502#44360502)

**4. What's the Gate? all 4 hexes or only the middle two?"** (Source: https://discord.com/channels/740870068178649108/1211591995328299038/1407983850327511040)
<img width="846" height="49" alt="image" src="https://github.com/user-attachments/assets/85f9741e-3bad-4ef3-b424-68c676e65092" />

**5. Initiative Token: Activate only one unit first or all?** (Source: https://discord.com/channels/740870068178649108/1211591995328299038/1407983083424186369)
<img width="857" height="86" alt="image" src="https://github.com/user-attachments/assets/1c472635-f985-4463-bd3b-04988bb375d7" />

DerSchlupf commented:
Sry Miss-click :/

DarthGandalf commented:
> 1) I prefer sort all the stuff by expansion.

done

> 2) we could mention adventure and skirmish mode as new game modes with link to game modes?

done

> 3) do we want move this part about empowered ability cards to expansion content

Not sure about this. It would be good to have empowered ability near empowered token+example. But it's also good to have empowered ability+stat to be near weak ability+stat for symmetry, in deck building. It's possible to move the token and example to Deck building section... e.g. between empowered ability and spells (+vertical art)

> 6) I would remove this one:

done. I'm not happy about the text though, needs more editing.

> I by the way like your solution

Thanks, but apparently that was wrong. The rewrite should clarify existing rules, not improve them (unless devs confirm new wording, ofc).

> 7) Does any asset of a stack token's other side exist?

Haven't seen yet. We may need to wait for the delivery to scan them.

-----

No strong opinion about the rest.

DarthGandalf commented:
Re layout: If we do move empowered ability example to deck building, the oceanid can go the same page in deck building. The 3 cove blocks could go to near creature banks where the empowered abilities example is now.

DarthGandalf commented:
The Initiative Token rules are confusing: as written here, this can be misunderstood as you need to pass the token even if only 1 player had units of this initiative

DerSchlupf commented:
> The Initiative Token rules are confusing: as written here, this can be misunderstood as you need to pass the token even if only 1 player had units of this initiative

Oh right. At the moment I'm a bit busy but I try to reword it in the next days. Thanks for scouting this 👍

DerSchlupf commented:
Qr-Codes to fandmade stuff added (picture in the first post), but...
- added also to digital build. i dont know how to add only to printable build. it is the question anyway, how we want to arrange this contents.
- pictures are in `assets/images` instead of `assets/qr-codes` cause my texstudio misrecognized the hyphen in "qr-codes" in the LaTeX code, and somehow it didn't work when the images were stored in the "QR code" folder. i moved them to "images"

DerSchlupf commented:
The new structure is in the first post. Some advices regarding that latest push:

1) I saved the images in box-covers, but I couldn't manage to create the new path in metadata.tex. I was getting error messages that I didn't understand. So the images are also saved in Images. If we leave them there, maybe you can still change the paths.

2) There exist no official picture of the tower box (maybe because it isnt a box for retail. i've made a picture with AI. If you zoom in and look closely you see diferences or discrepancies. I think the picture itself is okay.

3) Battlefield and naval battles introductions are together at one page. after that page follow the detailed rules about naval, and then battlefield. i think it is okay, that battlefield intro is NOT exactly before their detailed rules. 

4) please proofread all texts. the most of them is the same as before. but some introduction sentences are new.

DarthGandalf commented on sections/expansion_content.tex:
```diff
@@ -106,9 +106,11 @@ \subsection*{More of the same}
+    Contains the \textbf{Rampart Faction}: nature-oriented Units under the lead of Druids and Rangers.\par
     \medskip
-    With them comes a new Card type: The use of \pagelink{War Machines}{War Machines} support you permanent in your combats.
+    New game play mechanics:\par
```
```suggestion
    New gameplay mechanics:\par
```

DarthGandalf commented on sections/expansion_content.tex:
```diff
@@ -510,41 +517,52 @@ \subsection*{You haven't had enough yet?}
     \begin{minipage}{2.5cm}
         \includegraphics[width=\linewidth]{\qr/hero-creator.png}
     \end{minipage}
+    }
```
```suggestion
    }{}
```

Between the second {} is what happens when the toggle is false (non-printable)

DerSchlupf replied:
Oh thx for clarifying! Then I now have an idea how to make it, that it works.

DarthGandalf commented on sections/expansion_content.tex:
```diff
@@ -510,41 +517,52 @@ \subsection*{You haven't had enough yet?}
+    }
     \item Generate own custom Heroes and their unique Specialty with the  \href{https://k-adam.github.io/Homm3_hero_creator/}{Hero Creator} and let these Heroes write history.
+    \iftoggle{printable}{
     \end{minipage}
```
you can't interleave different kind of blocks: if should be fully within minipage, or vice versa

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -1,44 +1,69 @@
+\subsection*{\pagetarget{Miniatures}{Unit Miniatures}}
+The expansions make your Units appear even more alive.
+If you want to play your Combats with Miniatures, place and move them over the Combat Board instead of the Cards.
+You can put Miniatures on top of the Cards or keep the Cards next to the Combat Board forming an initiative bar for a better visualization of the order of activating Units.\par
```
```suggestion
You can put Miniatures on top of the Cards or keep the Cards next to the Combat Board to form an initiative bar for better visualization of the order in which Units activate.\par
```
Started late, I'll carry on tomorrow.

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -1,44 +1,69 @@
+      \item if you draw the any Unit you already have in your army,
+      \item if you draw any Unit from your Faction.
+    \end{itemize}
+  \item While you Recruit Neutral Units, you can Recruit neither any Units from a Faction that is controlled by a player nor any Units that are already in any player's army.
```
```suggestion
  \item While you Recruit Neutral Units, you cannot Recruit any Units from a Faction that is controlled by a player or any Units that are already in any player's army.
```

qwrtln replied:
Single negation beats double one.

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -1,44 +1,69 @@
-  Explained in \pagelink{War Machines}{Deck-building}.
-\end{expansion}
+\subsection*{Schools of Magic}
+\pagelink{Schools of Magic}{Schools of Magic} gain more significance with the expansions, because now there are Card effects which use them, e.g.~your Heroes can specialize in certain Schools to make their Spells more powerful.
```
```suggestion
\pagelink{Schools of Magic}{Schools of Magic} gain more significance with the expansions, as there are now Card effects which use them, e.g.~your Heroes can specialize in certain Schools to make their Spells more powerful.
```
This sound slightly more natural.

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -1,44 +1,69 @@
+  \end{wrapfigure}
+  Contains the \textbf{Fortress Faction}: poisonous swamp beasts under the lead of Witches and Beastmasters.\par
+  \medskip
+  New gameplay mechanics: The following new storytelling Card type.
```
```suggestion
  New gameplay mechanics: The new storytelling Card type described below.
```

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -74,122 +99,472 @@ \subsection*{Empowered Statistic Cards}
+   \medskip
+   New gameplay mechanics:\par
+   \smallskip
+   \pagelink{Elemental Map Tiles}{Elemental Map Tiles} aligned with Schools of Magic; magical \pagelink{Monolith Tokens}{Monoliths}, which allow your Heroes to traverse long distances on the Map; a new \pagelink{Elemental Damage}{type of damage} which ignores defense; and a unique Spell effect to \pagelink{Detailed Card Effects}{Summon Elemental Units (TODO: fix link)}, which support you in your  ...
```
```suggestion
   \pagelink{Elemental Map Tiles}{Elemental Map Tiles} aligned with Schools of Magic; magical \pagelink{Monolith Tokens}{Monoliths}, which allow your Heroes to traverse long distances on the Map; a new \pagelink{Elemental Damage}{type of damage} which ignores Defense; and a unique Spell effect to \pagelink{Detailed Card Effects}{Summon Elemental Units}, which support you in combat.  % TODO: fix link
```
Wasn't it supposed to be a comment?

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -74,122 +99,472 @@ \subsection*{Empowered Statistic Cards}
+    \begin{wrapfigure}{l}{0.35\textwidth}
+        \includegraphics[width=\linewidth]{\boxcovers/stronghold-box.png}
+    \end{wrapfigure}
+    Contains the \textbf{Stronghold Faction}: wild and orkish Units under the lead of Barbarians and Battle Mages.\par
```
```suggestion
    Contains the \textbf{Stronghold Faction}: wild and orkish Units under command of Barbarians and Battle Mages.\par
```
I think beasts and orcs are rather under command 😆

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -74,122 +99,472 @@ \subsection*{Empowered Statistic Cards}
+    \medskip
+    New gameplay mechanics:\par
+    \smallskip
+    \pagelink{Subterranean Map Tiles}{Subterranean Map Tiles}, which open a new world to explore and mystical \pagelink{Spell Scroll Cards}{Spell Scrolls},
+    which allow your Heroes to cast additional Spells, even if they are not so familiar with magic.
```
```suggestion
    \pagelink{Subterranean Map Tiles}{Subterranean Map Tiles}, which open a new world to explore, and mystical \pagelink{Spell Scroll Cards}{Spell Scrolls},
    which allow your Heroes to cast additional Spells even if they are not as familiar with magic.
```

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -74,122 +99,472 @@ \subsection*{Empowered Statistic Cards}
+    \end{wrapfigure}
+    Contains the \textbf{Tower Faction}: magic-affine Units under the lead of Wizards and Alchemists.\par
+    \medskip
+    This expansion doesn't introduce any new gameplay mechanics, but it expands on many existing components, e.g.~adds more \textbf{Heroes of other Factions}.
```
```suggestion
    This expansion doesn't introduce any new gameplay mechanics, but it expands many existing components, e.g.~it adds more \textbf{Heroes of other Factions}.
```

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -74,122 +99,472 @@ \subsection*{Empowered Statistic Cards}
-    \item \textbf{Hard}: 3 tokens
-    \item \textbf{Impossible}: 4 tokens
-  \end{itemize}
+\textbf{The following two expansions don't contain a new Faction, but they offer a lot of new gameplay mechanics or even new interesting game modes:}
```
```suggestion
\textbf{The following two expansions don't contain a new Faction, but they offer many new gameplay mechanics or even new game modes:}
```
Again, this is judgemental. We should be objective and factual here.

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -74,122 +99,472 @@ \subsection*{Empowered Statistic Cards}
+        \begin{wrapfigure}{r}{0.2\linewidth}
+            \includegraphics[width=\linewidth]{\images/stack_token.png}
+        \end{wrapfigure}
+    Each Stack Token modifies a Unit's statistics in the following way: +1~\svg{attack}, +1~\svg{defense}, +1~\svg{health_points}, or +2~\svg{initiative}.
```
```suggestion
    Each Stack Token modifies a Unit's statistic in one of the following ways: +1~\svg{attack}, +1~\svg{defense}, +1~\svg{health_points}, or +2~\svg{initiative}.
```
Or this.

DarthGandalf replied:
probably because it conflicted with another suggestion with this or nearby lines which you accepted instead

this can be resolved manually with text editor

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -74,122 +99,472 @@ \subsection*{Empowered Statistic Cards}
+            \includegraphics[width=\linewidth]{\images/stack_token.png}
+        \end{wrapfigure}
+    Each Stack Token modifies a Unit's statistics in the following way: +1~\svg{attack}, +1~\svg{defense}, +1~\svg{health_points}, or +2~\svg{initiative}.
+    Furthermore, each Unit with a Stack Token is now a \textbf{Stacked Unit}, which is similar to reinforced Faction Units: when it takes damage equal to or greater than its HP, instead of removing the Unit Card from the Combat Board, discard the Stack Token from that Unit and deal any leftover damage (if any), deducting it from the new HP.
```
```suggestion
    Furthermore, each Unit with a Stack Token is now a \textbf{Stacked Unit}, which is similar to reinforced Faction Units: when it takes damage equal to or greater than its \svg{health_points}, instead of removing the Unit Card from the Combat Board, discard the Stack Token from that Unit and deal any leftover damage, deducting it from the new \svg{health_points}.
```
I think this text was already merged before, but "if any" is redundant here.

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -74,122 +99,472 @@ \subsection*{Empowered Statistic Cards}
+
+    \subsection*{Gained Stacked Units}
+    Some rewards allow you to gain a Stacked Creature Bank Unit Card.
+    Track which of your Units are Stacked by whichever means comfortable, e.g.~by remembering that or by having some token or cube on it.
```
```suggestion
    Track which of your Units are Stacked by whatever means is comfortable, e.g.~by remembering it or by placing a Token or Cube on it.
```

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -74,122 +99,472 @@ \subsection*{Empowered Statistic Cards}
+\vspace*{1em}
+\begin{expansion}[before=\vspace*{-11mm}]{navalbattles}
+    \subsection*{\pagetarget{Empowered Ability Token}{Empowered Ability Cards}}
+    This new type of Cards improves your Heroes Ability Cards and it's explained in \pagelink{Empowered Ability}{Deck-building}.
```
```suggestion
    This new type of Card improves your Hero's Ability Cards and is explained in \pagelink{Empowered Ability}{Deck-building}.
```

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -74,122 +99,472 @@ \subsection*{Empowered Statistic Cards}
+    If an Empowered Ability Card isn't available, you can't spend the Empowered Ability Token and must keep it for later turns.
+    \vspace*{1em}
+
+    You can gain Empowered Ability Tokens in certain Scenarios only (e.~g.~``Island of Fire'' from the Naval Battles Mission Book.).\\
```
```suggestion
    You can gain Empowered Ability Tokens only in certain Scenarios (e.~g.~``Island of Fire'' from the Naval Battles Mission Book).\\
```

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -74,122 +99,472 @@ \subsection*{Empowered Statistic Cards}
+\begin{expansion}{battlefield}
+  \begin{multicols*}{2}
+  \subsection*{\pagetarget{Battlefield Combat}{Battlefield Board in regular games}}
+  We recommend using the Battlefield Combat Board only for \textbf{Player vs.~Player Combats}, even though it is also possible to use it in Neutral Combats.
```
```suggestion
  It is recommended to use the Battlefield Combat Board only for \textbf{Player vs.~Player Combats}, even though it is also possible to use it in Neutral Combats.
```
Who's we? Smith & Wesson and me?

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -74,122 +99,472 @@ \subsection*{Empowered Statistic Cards}
+
+  \subsection*{Rules Changes in Combat Setup}
+  \begin{itemize}
+    \item \textbf{Place obstacles} – Starting with the attacking player, both players take turns chosing and placing one Obstacle Token on the Battlefield Board until \textbf{all Obstacle Tokens are placed}. No Obstacle can be adjacent to another Obstacle or to any player's deployment zone.
```
```suggestion
    \item \textbf{Place obstacles} – Starting with the attacking player, both players take turns choosing and placing one Obstacle Token on the Battlefield Board until \textbf{all Obstacle Tokens are placed}.
    No Obstacle can be adjacent to another Obstacle or to any player's deployment zone.
```

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -74,122 +99,472 @@ \subsection*{Empowered Statistic Cards}
+    The player with the Token activates first \textbf{one of his Units of the same Initiative value}.
+    Then, the players alternate in activating all of their Units of that same Initiative value one by one.
+    Once all Units with that Initiative value are activated, pass the Initiative Token to the other player, who keeps it until initiative ties must be resolved again in the same way.
+    \item \textbf{Gate Token} – During a Siege, defending Units may enter the Gate Token \textbf{only at the two middle Hexes}, where the Gate is printed on.
```
```suggestion
    \item \textbf{Gate Token} – During a Siege, defending Units may enter the Gate Token \textbf{only at the two middle Hexes} where the Gate is printed.
```

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -74,122 +99,472 @@ \subsection*{Empowered Statistic Cards}
+  \end{itemize}
+  \subsection*{Rules Changes during Combat}
+  \begin{itemize}
+    \item \textbf{Unit Movement} – Each Unit's Movement is equal to its Initiative value: a Unit of Initiative 8 can move up to 8 Hexes.
```
```suggestion
    \item \textbf{Unit Movement} – Each Unit's Movement is equal to its Initiative value: a Unit of Initiative 8 can move up to 8 hexes.
```
Not a game component, I think?

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -74,122 +99,472 @@ \subsection*{Empowered Statistic Cards}
+\subsection*{Stretch Goal Content}
+
+Four boxes of stretchgoal content have been released for the boardgame; they are shown at the bottom of this page.\par
+These expansions primarily contains aesthetic components (like miniatures of neutral Units or Map Locations), but they also extend the variety of existing components (like additional Artifact or Spell Cards or new Hero Cards for all Factions).\par
```
```suggestion
These expansions primarily contain aesthetic components (like Miniatures of Neutral Units or Map Locations), but they also extend the variety of existing components (like additional Artifact or Spell Cards or new Hero Cards for all Factions).\par
```

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -74,122 +99,472 @@ \subsection*{Empowered Statistic Cards}
+Take a look at the \textbf{fanmade expansions and contents} from the \textbf{HoMM3 BG Community}. They are still work in progress. Feel free to contribute.
+  \begin{itemize}
+    \begin{minipage}{5cm}
+      \item The \textbf{Fanmade Mission Book} contains many well playtested Scenarios with unique rules for even more spice in your games:
```
```suggestion
      \item The \textbf{Fan-made Mission Book} contains many well-playtested Scenarios with unique rules for even more variety in your games:
```

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -74,122 +99,472 @@ \subsection*{Empowered Statistic Cards}
+    \end{minipage}\par
+    \smallskip
+    \begin{minipage}{5cm}
+      \item The \textbf{Fanmade Factory Expansion} introduces the beloved missing Faction.
```
```suggestion
      \item The \textbf{Fan-Made Factory Expansion} introduces the missing Faction.
```
"Beloved" is too judgemental for a rule book.

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -74,122 +99,472 @@ \subsection*{Empowered Statistic Cards}
+  \end{center}
+  \begin{itemize}
+    \begin{minipage}{5cm}
+        \item Bring your creative ideas to life by building own Scenarios with the help of the \textbf{Scenario Map Editor}:
```
```suggestion
        \item Bring your creative ideas to life by building your own Scenarios with the help of the \textbf{Scenario Map Editor}:
```
It was discussed already, but isn't the subject missing now?

qwrtln replied:
Satzgegenstand.

DerSchlupf commented on sections/expansion_content.tex:
```diff
@@ -74,122 +99,472 @@ \subsection*{Empowered Statistic Cards}
+ \end{expansion}
+ \vspace*{\fill}
+ \columnbreak
+\subsection*{You haven't had enough yet?}
```
What do you prefer? We can take another header. But I do not care. I like them all :)

qwrtln replied:
All right, let's stick with this one but without "You".

DerSchlupf commented on sections/expansion_content.tex:
```diff
@@ -74,122 +99,472 @@ \subsection*{Empowered Statistic Cards}
+    Use it only to break any Initiative ties of opposing Units:
+    The player with the Token activates first \textbf{one of his Units of the same Initiative value}.
+    Then, the players alternate in activating all of their Units of that same Initiative value one by one.
+    Once all Units with that Initiative value are activated, pass the Initiative Token to the other player, who keeps it until initiative ties must be resolved again in the same way.
```
What do you mean?

ExeVirus commented on sections/expansion_content.tex:
```diff
@@ -1,44 +1,68 @@
+  \begin{wrapfigure}{l}{0.35\textwidth}
+      \includegraphics[width=\linewidth]{\boxcovers/fortress-box.png}
+  \end{wrapfigure}
+  Contains the \textbf{Fortress Faction}: poisonous swamp beasts under the lead of Witches and Beastmasters.\par
```
Capitalize: `Poisonous swamp...`

qwrtln replied:
Ah, the flip-flop is my fault. I suggested "under command of" in one place but not in the others. @DerSchlupf, do you think we should change it everywhere?

ExeVirus commented on sections/expansion_content.tex:
```diff
@@ -51,66 +75,189 @@ \subsection*{Empowered Statistic Cards}
+  \begin{multicols}{2}
+    \begin{itemize}
+    \item[\textbf{1.}] Name
+    \item[\textbf{2.}] Fluff
```
`Fluff` -> `Flavor Text`

qwrtln replied:
Fluff is actually taken straight from the original. You think flavor text is better?

<img width="752" height="300" alt="image" src="https://github.com/user-attachments/assets/f6a07a77-6d6d-4f85-8436-692d81a46eb4" />

DarthGandalf replied:
Flavor text is the common term for this thing in other games

qwrtln replied:
I noticed that translations tend to prefer it too. Shall we change it everywhere?

qwrtln replied:
It's only in one other place. I'll change it.

ExeVirus commented on sections/expansion_content.tex:
```diff
@@ -51,66 +75,189 @@ \subsection*{Empowered Statistic Cards}
+  \begin{wrapfigure}{l}{0.35\textwidth}
+    \includegraphics[width=\linewidth]{\boxcovers/rampart-box.png}
+  \end{wrapfigure}
+  Contains the \textbf{Rampart Faction}: nature-oriented Units under the lead of Druids and Rangers.\par
```
Capitalize: `Nature-oriented...`

qwrtln replied:
For future's sake, for simple edits like that, please use suggestions:
<img width="752" height="300" alt="Screenshot From 2025-10-07 21-30-18" src="https://github.com/user-attachments/assets/49e05d05-44f6-4530-a332-0ba1288a5bfb" />
```suggestion
  Contains the \textbf{Rampart Faction}: Nature-oriented Units under the lead of Druids and Rangers.\par
```

ExeVirus commented on sections/expansion_content.tex:
```diff
@@ -51,66 +75,189 @@ \subsection*{Empowered Statistic Cards}
+  \begin{wrapfigure}{l}{0.35\textwidth}
+    \includegraphics[width=\linewidth]{\boxcovers/conflux-box.png}
+  \end{wrapfigure}
+  Contains the \textbf{Conflux Faction}: magical Creatures of all elements under the lead of Elementalists and Planeswalkers.\par
```
Capitalize: `Magical Creatures...`

ExeVirus commented on sections/expansion_content.tex:
```diff
@@ -51,66 +75,189 @@ \subsection*{Empowered Statistic Cards}
+  \begin{wrapfigure}{l}{0.35\textwidth}
+    \includegraphics[width=\linewidth]{\boxcovers/stronghold-box.png}
+  \end{wrapfigure}
+  Contains the \textbf{Stronghold Faction}: wild and orkish Units under command of Barbarians and Battle Mages.\par
```
Capitalize: `Wild and...`

ExeVirus commented on sections/expansion_content.tex:
```diff
@@ -51,66 +75,189 @@ \subsection*{Empowered Statistic Cards}
+  \begin{wrapfigure}{l}{0.35\textwidth}
+    \includegraphics[width=\linewidth]{\boxcovers/tower-box.png}
+  \end{wrapfigure}
+  Contains the \textbf{Tower Faction}: magic-affine Units under the lead of Wizards and Alchemists.\par
```
Capitalize, and change 'affine' to: `Magic-focused`

ExeVirus commented on sections/expansion_content.tex:
```diff
@@ -150,46 +297,284 @@ \subsection*{Empowered Statistic Cards}
+  \medskip
+  \subsection*{Battlefield Board in Neutral Combats}
+  If you decide to use the Battlefield Board in Combats with Neutral Units, ignore the Round Limit:
+  at the end of each Combat Round you may freely decide to Retreat or to gain an additional Combat Round with no need to spend a MP \svg{movement}.
```
Capitalize the `At the....`

ExeVirus commented on sections/expansion_content.tex:
```diff
@@ -150,46 +297,284 @@ \subsection*{Empowered Statistic Cards}
+  \end{figure}
+  \vspace*{1em}
+  \note{3}{
+    All Unit Miniatures also count as Obstacles.
```
`All Unit Miniatures block like Combat Obstacles`

I assume they aren't actually able to be removed with "Remove Obstacle" spells or similar.

DerSchlupf replied:
We should keep the current wording in this case, cause it's an official rule that they count as obstacles. Regarding remove Obstacle that spell explicitly forbid removing units.

ExeVirus commented on sections/expansion_content.tex:
```diff
@@ -150,46 +297,284 @@ \subsection*{Empowered Statistic Cards}
+ \columnbreak
+\subsection*{Haven't had enough yet?}
+Take a look at the \textbf{fan-made expansions and content} from the \textbf{HoMM3 BG Community}.
+They are still works in progress.
```
Many are play-ready, and many more are a work in progress.

ExeVirus commented on sections/expansion_content.tex:
```diff
@@ -150,46 +297,284 @@ \subsection*{Empowered Statistic Cards}
+\begin{multicols}{4}
+    \begin{center}
+        \includegraphics[width=0.8\linewidth]{\boxcovers/stretchgoals-factionunits.png}\newline
+        \footnotesize Core game's Faction Miniatures\par
```
Might as well capitalize `Game's` like the other words here.

ExeVirus commented on sections/expansion_content.tex:
```diff
@@ -150,46 +297,284 @@ \subsection*{Empowered Statistic Cards}
+        \footnotesize Core game's Faction Miniatures\par
+        \columnbreak
+        \includegraphics[width=0.8\linewidth]{\boxcovers/stretchgoals-neutral.png}\newline
+        \footnotesize \nth{1} Campaign's Miniatures\par
```
`Campaign's Miniatures` -> `Campaign Miniatures`

qwrtln replied:
Fixed.

ExeVirus commented on sections/expansion_content.tex:
```diff
@@ -150,46 +297,284 @@ \subsection*{Empowered Statistic Cards}
+        \footnotesize \nth{2} Campaign's gameplay components\par
+        \columnbreak
+        \includegraphics[width=0.8\linewidth]{\boxcovers/stretchgoals-crystal.png}\newline
+        \footnotesize \nth{2} Campaign's Miniatures\par
```
`Campaign Miniatures`

qwrtln replied:
Fixed.

ExeVirus commented on assets/gimp-files/arrow-curved-red.png:
These PNGs are huge - seems like a waste of space. At least compress them with https://squoosh.app/ or similar.

For example, this file lossless compresses from 18.7KB -> 2.67KB

<img width="608" height="608" alt="arrow-curved-red" src="https://github.com/user-attachments/assets/680b8a73-3644-49f7-ab6b-073b3bee8e4e" />

Let me know if you want help with automated compression/Image texture cleanup at some point - I do this a lot.

qwrtln commented on sections/expansion_content.tex:
```diff
@@ -150,46 +297,284 @@ \subsection*{Empowered Statistic Cards}
+
+  \textit{Bob happens to have an Empowered Ability Token.
+    He may either use it, or save the token for later.
+    He decides to use it, so he removes the ``Offense'' Ability Card and take the Empowered ``Offense'' Ability Card instead.}
```
Removing is different from discarding, it is correct here.
